src/redux/feedbacks/operations.ts

```diff
@@ -15,7 +32,16 @@ export const fetchReviews = createAsyncThunk<Review[], FetchReviewsParams>(
       const response = await axiosInstance.get(
         `api/reviews/?limit=${limit}&offset=${offset}`
       )
-      return response.data
+      const reviews: Review[] = response.data
+      const handledReviews = await Promise.all(
+        reviews.map(async (review) => {
+          const userResponse = await thunkAPI.dispatch(
```

Comment:
you don't have `user_name` in `api/reviews`?
maybe the backend should store this value and return it, let's discuss this at the meeting